doc/cpp/cable_cell.rst

```diff
@@ -377,6 +377,8 @@ Overriding properties locally
    the morphology.
 
 
+.. _cable-cell-probes:
```

Comment:
Would this be the content that's been put under `simulation`? I'm not averse to this, but maybe we could make the concept docs a different task, as it's not exactly in scope for this PR, and would delay merging.

Reply:
I don't know what you are referring to by "under simulation".

The issue here is that documentation has been created under Python API, but nothing added to concepts. The cpp and python docs might be simplified if a lot of the common part of that documentation is moved to concepts. So I think that it is actually in the scope of this PR.

If it is in a separate PR, then it should be done ASAP, otherwise it will be forgotten.

Reply:
I mean, I added some text in the documentation of the `simulation` class that explained the probes and sampling. Is this suitable material for moving into Concepts?